package.json

```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "scripts": {
     "sass:watch": "gulp watch",
+    "gulp:watch": "gulp-watch",
```

Comment:
This doesn't work! Is the intention to split up sass and JS watching?

With this `npm run sass:watch` will also watch and build JS, but that's probably not what we want?

Reply:
I fixed the bug, and changed sass:watch to only watch the sass files, and made gulp-watch to do watch both js and sass.

I also updated run.sh and config.yml to build the js in addition to the sass.

Reply:
If we're building JS on docker run and deploys, we don't need to check in the minified JS, right? This would be similar to how we don't currently check in the contents of `static/css/compiled` to source control.

Reply:
That tracks.  I will update.

Reply:
Updated.